aws_lambda_builders/workflow.py

```diff
@@ -22,6 +24,22 @@
 Capability = namedtuple('Capability', ["language", "dependency_manager", "application_framework"])
 
 
+# TODO: Move sanitize out to its own class.
+def sanitize(func):
+    """
+    sanitize the executable path of the runtime specified by validating it.
+    :param func: Workflow's run method is sanitized
+    """
+
+    @functools.wraps(func)
+    def wrapper(*args, **kwargs):
```

Comment:
Should we pass in self here and make that an explicit argument instead of getting the first value out of args?

Reply:
Sure we can do that.